package.json

```diff
@@ -5,28 +5,28 @@
   "version": "1.0.11",
   "license": "MIT",
   "scripts": {
-    "test": "jest --env=jsdom --browser",
+    "test": "jest --env=node",
     "start": "ts-node src/index.ts",
-    "build": "tsc -p tsconfig.json --outDir lib",
+    "install": "tsc -p tsconfig.json --outDir lib",
     "examples:types": "ts-node example/types.ts",
     "examples:references": "ts-node example/references.ts",
     "prepublish": "yarn build && npm-auto-version",
     "postpublish": "git push origin --tags"
   },
   "dependencies": {
-    "firebase": "^5.8.1",
-    "graphql": "^14.1.1",
-    "graphql-tag": "^2.10.1"
+    "firebase": "^8.7.1",
+    "graphql": "^15.5.1",
+    "graphql-tag": "^2.12.5"
   },
   "devDependencies": {
```

Comment:
A dev dependency on prettier should be added right? `npm i -D prettier`

Reply:
oh should I save that? okay.